GUI/src/app/pages/generator/generator.component.ts

```diff
@@ -195,6 +196,25 @@ export class GeneratorComponent implements OnInit {
       return;
     }
 
+    let noLogicConfirmed = localStorage.getItem("noLogicConfirmed");
+    if ((!noLogicConfirmed || noLogicConfirmed == "false") && this.global.generator_settingsMap["logic_rules"] === "none") {
```

Comment:
Local storage isn't available depending on browser settings and in some privacy focused niche browsers. This causes any calls to the class to throw an error. Hence this whole block should be wrapped in a try/catch to mimick other instances where local storage is used. In case of catch the GUI should simply advance with no warning imo (empty catch).